sklearn/discriminant_analysis.py

```diff
@@ -425,20 +425,6 @@ def fit(self, X, y, store_covariance=None, tol=None):
         y : array, shape (n_samples,)
```

Comment:
We don't actually have a policy on that, do we? But yeah, would be better I guess.